app/src/screens/Proposals.js

```diff
@@ -286,23 +242,26 @@ const ProposalInfo = ({
   myStakes,
   requestToken,
   selectProposal = false,
-}) => (
-  <div
-    css={`
-      width: ${23 * GU}px;
-    `}
-  >
-    {selectProposal && (
-      <IdAndTitle {...proposal} selectProposal={selectProposal} />
-    )}
-    <ConvictionBar proposal={proposal} withThreshold={requestToken} />
-    {myStakes.has(proposal.id) && (
-      <Tag>
-        {`✓ Supported: ${myStakes.get(proposal.id)} ${stakeToken.tokenSymbol}`}
-      </Tag>
-    )}
-  </div>
-)
+}) => {
+  const myStakeInfo = myStakes.find(stake => stake.proposal === proposal.id)
+  return (
+    <div
+      css={`
+        width: ${23 * GU}px;
+      `}
+    >
+      {selectProposal && (
+        <IdAndTitle {...proposal} selectProposal={selectProposal} />
+      )}
+      <ConvictionBar proposal={proposal} withThreshold={requestToken} />
+      {myStakeInfo && (
+        <Tag>
+          {`✓ Supported: ${myStakeInfo.amount} ${stakeToken.tokenSymbol}`}
```

Comment:
I think this should be `stakedAmount` ? 
```suggestion
          {`✓ Supported: ${myStakeInfo.stakedAmount} ${stakeToken.tokenSymbol}`}
```